plans/importer.py

```diff
@@ -1,4 +1,8 @@
 def import_name(name):
-    components = name.split('.')
-    mod = __import__('.'.join(components[0:-1]), globals(), locals(), [components[-1]] )
-    return getattr(mod, components[-1])
+    """ import module given by str or pass the module if it is not str """
+    if type(name) == str:
```

Comment:
Couldn't this be written as `isinstance(name, str)` ?

Reply:
Fixed